en/docs/get-started/integration-quick-start-guide.md

```diff
@@ -154,25 +156,44 @@ The following project files and executable back-end services are available in th
 
 - **Backend**: This contains an executable .jar file that contains mock back-end service implementations for the Pine Valley Hospital and Grand Oak Hospital.
 
+- **bin**: This contains a script to copy artifacts and run the backend service.
+
 ### Step 2 - Running the integration artifacts
 
 Follow the steps given below to run the integration artifacts we developed on a Micro Integrator instance that is installed on a VM.
 
-1. Start the back-end services.
+1. Run `run.sh/run.bat` script in `<mi-qsg-home>/bin` based on the OS to up the workspace.
+    1. Open a terminal and navigate to the `<mi-qsg-homeE>/bin` folder.
+    2. Execute the relevant command:
```

Comment:
This step contains too much information. Let's break this down into multiple steps. The content on starting the backend services and deploying the healthcare service should probably be steps of their own or sub-steps under a properly named umbrella. Right now this talks about executing a command but we should say what we're trying to do in the step itself.